Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp

```diff
@@ -456,4 +454,32 @@ void MolDraw2DSVG::outputClasses() {
   }
   d_os << "' ";
 }
+
+template <class t_obj> void MolDraw2DSVG::outputTagClasses(const t_obj *obj) const {
```

Comment:
I've been trying to avoid adding methods (even private methods) to classes that don't really need to be in the class. (Note: the class `MolDraw2D` is a really bad example here, since it contains a bunch of methods that should be functions).

It seems like `outputTagClasses()` and `outputMetaData()` don't actually need to be methods of `MolDraw2DSVG`. The only class data which they use is the output stream, so they could both be free functions which take an `std::ostream &` as an argument along with the `const t_obj *`.

Reply:
Those functions use the output stream (d_os) and active classes (d_activeClass). So we are exposing almost all private members of the class. I've changed the code but I feel we are getting into diminishing returns here...

Reply:
thanks